apps/blockchain/lib/blockchain/contract/message_call.ex

```diff
@@ -56,7 +57,7 @@ defmodule Blockchain.Contract.MessageCall do
   """
   @spec execute(t()) :: {EVM.state(), EVM.Gas.t(), EVM.SubState.t(), EVM.VM.output()}
   def execute(params) do
-    fun = get_fun(params.recipient)
+    run = EVM.MessageCall.get_run_function(params.recipient)
```

Comment:
small remark: you defined alias for `EVM.MessageCall` but you're using full module name